### ISSUE TITLE
RTX support planned...

### ISSUE THREAD

**Issue by elviras9t** [world8th/vRt#7]
From November should to beginning development of "Crossroads" project, which includes RTX support too... Any ideas, proposals, issues you can add here.

**Comment by oscarbg** [comment]
Hi,
Nice project& effort..
Have a Titan V and seems you have added NV raytracing ext support very recently.. as Titan V supports that ext. I have some questions:
1)if I clone project & build right now, will use NV RT ext on Titan V? I.e. is support activated transparently in case ext. is detected or is activated by a GPU name.. in that case any way to force using non NV ext. I.e. compute shaders “fallback” mode (ie some command line flag)? 
This could be useful for comparing perf. using NV ext. vs not..
Also would nice if in console output we get some info on if using NV VK ext or not?
2)NV RT ext Spv &glsl & spec. has gone very recently from NVX to NV adding new callable shaders seems.. still no driver yet, but hope soon.. are callable shaders useful for vRt for easier impl. or better perf? In any case hope Final ext support is added ASAP as drivers appear..
Some links I tweeted https://twitter.com/oscarbg81/status/1057765861170257923?s=21
3) finally the most controversial request and probably an Herculean effort, is if it would make sense to expose you library as a Vulkan layer that exposes NV Vulkan raytracing ext. on non supported GPUs.. think of it, of kind of “DXR fallback layer” for VK NV RT ext. .. I hope request is clear..

**Comment by elviras9t** [comment]
1. Today, yes, can automatically detect (but we added RTX to Turing's defaultly), and yes, we support non-RTX graphics cards by our historical purpose reasons.
2. Support in glslang still need await to support, but we move on as soon as possible.
3. Early we named as "layer", today we just naming as "library".
We still considering about adding support of command-line flag to disable or enable RTX support. Also, vRt library implements RTX mode by extension (named as `vRtX`), so still need some handy work to add support HW acceleration (mainly, in correct Instancing connection).
